lib/attributor/types/csv.rb

```diff
@@ -29,5 +29,9 @@ def self.example(context=nil, options: {})
       return collection.join(',')
     end
 
+    def self.family
```

Comment:
But the flip side is, it will *always* be a string when rendered, and never an array.

The only time you see it as an array is as a loaded value in code.